lib/matplotlib/tests/test_colors.py

```diff
@@ -1,10 +1,10 @@
 import copy
 import itertools
-
 import numpy as np
 import pytest
 
-from numpy.testing import assert_array_equal, assert_array_almost_equal
+from numpy.testing import (assert_array_equal, assert_array_almost_equal,
+                           assert_equal)
```

Comment:
I think we've generally moved away from using `assert_equal`. Especially because you're just comparing single values, just assert that the two things are equal.